examples/onnxruntime/training/text-classification/run_glue.py

```diff
@@ -188,6 +189,12 @@ class ModelArguments:
             "with private models)."
         },
     )
+    use_ort: bool = field(
+        default=True,
+        metadata={
+            "help": "disable onnxruntime to accelerate training"
+        },
+    )
```

Comment:
yes, it's added for benchmarking purpose.
do you have suggestion for benchmarking? we just want to make sure the torch baseline is same as ort's run. if use example in HF repo, will it be possible that the example is diff with optimum repo?

Reply:
It depends on how you are going to calculate the latency and throughput. If you want to leverage the [inner timer](https://github.com/huggingface/transformers/blob/main/src/transformers/trainer.py#L1782) of Trainers, as `ORTTrainer` subclassed `Trainer`, running those two scripts should not have any influence.

Here I prefer that we only keep the `ORTTrainer` for training to keep its consistency with transformers and to not distract users with extra flags or choices, but you can add a link to the transformers example in README in case users want to compare with PyTorch.

Reply:
got you point, thx!
one concern from me is that we need to make sure optimum example is as new as HF example, for example if HF example got updated then optimum example needs same changes, otherwise the benchmark result may be wrong.

Reply:
That's a good point - how can we keep codes in Optimum always aligned transformers. We will probably develop a tool for this. But by all means, adding the tag for using `transformers.Trainer` won't help avoid this problem, in general, the examples are stable and backward compatible.
So here I prefer that we remove the tag for choosing ORTTrainer/Trainer.